packages/common-server/src/template.ts

```diff
@@ -65,6 +73,52 @@ function genDefaultContext(targetNote: NoteProps) {
   };
 }
 
+let _INIT_HELPERS = false;
+
+class TemplateHelpers {
+  static init() {
+    _.map(this.helpers, (v, k) => {
+      Handlebars.registerHelper(k, v);
+    });
+  }
+
+  static helpers = {
```

Comment:
from my understanding of how this works, these property names are important in that the user needs to use the exact same string in their template for it to work.  Can we add a comment for caution as changing these property names is a breaking change?